specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21567,6 +21567,415 @@
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Creates the artifacts for web site, or a deployment slot.",
+        "description": "Description for Creates the artifacts for web site, or a deployment slot.",
+        "operationId": "WebApps_DeployWorkflowArtifacts",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Artifacts deployed."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Deploys workflow artifacts": {
+            "$ref": "./examples/PostDeployWorkflowArtifacts.json"
+          },
+          "Delete workflow artifacts": {
+            "$ref": "./examples/DeleteDeployWorkflowArtifacts.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/deployWorkflowArtifacts": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Creates the artifacts for web site, or a deployment slot.",
+        "description": "Description for Creates the artifacts for web site, or a deployment slot.",
+        "operationId": "WebApps_DeployWorkflowArtifactsSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of the deployment slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Artifacts deployed."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Deploys workflow artifacts": {
+            "$ref": "./examples/PostDeployWorkflowArtifacts.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
>true [](http://example.com/codeflow?start=39&length=4)

needs to include 202

Reply:
Updated the tag to reflect that this is not a long running operation